aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -78,42 +81,90 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> Default
 }
 
 impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGroupValue<T, V> {
-    fn set_base_values(
-        &mut self,
-        shifted_idx: ShiftedTxnIndex,
-        // TODO[agg_v2](fix) add layout to values
-        values: impl IntoIterator<Item = (T, V)>,
-    ) {
+    fn set_raw_base_values(&mut self, values: impl Iterator<Item = (T, V)>) {
+        let shifted_idx = ShiftedTxnIndex::zero();
```

Comment:
nit: calling this zero_idx, shifted_zero, shifted_zero_idx, or basically anything indicating it's 0 index can make it marginally more readable imho.

Reply:
Done.